docs/requirements.txt

```diff
@@ -1,12 +1,6 @@
-ipykernel
 matplotlib
-nbsphinx==0.8.12
-nbconvert
-numpy
-scikit-learn >=0.24.0
+pandas
```

Comment:
Do the examples no longer require scikit-learn? I think it'd be good to have the explicit requirement.

Reply:
They do but we have scikit-learn as an explicit dependency. I don't see a point of adding this here again.